lib/fluent/plugin/statusz.rb

```diff
@@ -12,20 +12,114 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require 'erb'
+
 # Module for collecting diagnostic information and formatting it as an
 # HTML page to serve on the /statusz endpoint.
 module Statusz
   module_function
 
-  def response
-    # TODO(davidbtucker): Add more status information here.
-    [
-      '<html>',
-      '<body>',
-      '<h1>Status</h1>',
-      "<b>Command-line flags:</b> #{CGI.escapeHTML(ARGV.join(' '))}",
-      '</body>',
-      '</html>'
-    ].join("\n") + "\n"
+  def response(plugin)
```

Comment:
Where is `plugin` used?

Reply:
In the template.

Reply:
Can you please add a comment before `ERB.new(STATUSZ_TMPL).result(binding)` to that effect?

Reply:
Just floating this up for attention.

Reply:
Oops, done.